tests/test_model.py

```diff
@@ -228,6 +228,29 @@ def test_call_concat(model, outputs, expected):
             np.testing.assert_equal(y, expected)
 
 
+@pytest.mark.parametrize(
+    'device, num_workers',
+    [
+        ('cpu', None),
+        ('cuda:0', None),
+        ('cpu', 1),
+        ('cuda:0', 1),
+        ('cpu', 2),
+        ('cuda:0', 2),
```

Comment:
Do you know what `onnxruntime` does when you select a GPU, but also specify `num_workers > 1`. Is it ignored or does it still use multiple CPUs for other (whatever this might be) processing?

Reply:
Good question, I just tried it out while monitoring the CPU percentage of the current process, and it made no difference what `num_workers` was set to when running on the GPU. So it seems to be ignored.

Reply:
Great, this sounds like good behavior and fits well with your description "number of threads for running onnxruntime inference on cpu."